### PR TITLE
Organize extensions list by category.

### PR DESCRIPTION
**Pull request by yunelee**
- List extensions by category.
- Add extensions to list (amp-form, amp-live-list).

For https://github.com/ampproject/docs/issues/58

### CODE REVIEW

**Comment by erwinmombay** [comment]
this is awesome!

**Comment by erwinmombay** [comment]
this LGTM, looping in @jmadler and @ericlindley-g  for further feedback

File: extensions/README.md [hunk 1]
```diff
@@ -6,51 +6,106 @@ AMP Extensions are either extended components or extended templates.
...
-| [`amp-kaltura-player`](amp-kaltura-player/amp-kaltura-player.md) | Displays the Kaltura Player as used in [Kaltura's Video Platform](https://corp.kaltura.com/). |
 | [`amp-lightbox`](amp-lightbox/amp-lightbox.md) | Allows for a “lightbox” or similar experience. |
-| [`amp-list`](amp-list/amp-list.md) | Dynamically downloads data and creates list items using a template. |
 | [`amp-mustache`](amp-mustache/amp-mustache.md) | Allows rendering of [`Mustache.js`](https://github.com/janl/mustache.js/) templates. |
-| [`amp-pinterest`](amp-pinterest/amp-pinterest.md) | Displays a Pinterest widget or Pin It button. |
-| [`amp-reach-player`](amp-reach-player/amp-reach-player.md) | Displays a [Beachfront Reach](https://beachfrontreach.com/) video player. |
 | [`amp-sidebar`](amp-sidebar/amp-sidebar.md) | Provides a way to display meta content intended for temporary access such as navigation, links, buttons, menus. |
+| [`amp-user-notification`](amp-user-notification/amp-user-notification.md) | Displays a dismissable notification to the user. |
+
+### Scripts
```
Comment (erwinmombay) [inline extensions/README.md]: maybe we can have a more descriptive section for the service worker stuff?

cc @ericlindley-g

**Comment by ericlindley-g** [comment]
LGTM—this is great!! @rudygalfi as FYI for this nice component organization change

I agree, and could see a more descriptive serviceworker blurb (maybe even with a link to a neutral description like [w3's](https://www.w3.org/TR/service-workers/)). Also worth distinguishing `amp-list` and `amp-live-list` better, and making tweaks to a number of others.

However, I think those could be part of a separate documentation pass. This organization is a great first step, so I wouldn't hold it up based on other tweaks (unless they're ready to go)

**Comment by rudygalfi** [comment]
Awesome!

**Comment by erwinmombay** [comment]
@yunelee mind rebasing? LGTM. will merge when green again

**Comment by yunelee** [comment]
Will work on better blurbs next!

I attempted to rebase, but am new to Git, so feel free to let me know if I need to do things differently.

**Comment by erwinmombay** [comment]
@yunelee woot, looks like the rebase was bad :( will try to see if its easily fixable and i can give you the command to run.

**Comment by erwinmombay** [comment]
@yunelee so looks like its pretty difficult now to fix this because i think you did a merge. i can cherry pick your changes into a PR and fix it there while maintaining your authorship, if that is ok with you. I'll have you as the reviewer

**Comment by yunelee** [comment]
@erwinmombay Augh, sorry about that. Yes, feel free to cherry pick, or let me know if there's something I can do to make less work for you!

**Comment by erwinmombay** [comment]
landed as https://github.com/ampproject/amphtml/pull/4036

**Pull request closed without merging**